<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QProgramUI</class>
 <widget class="QWidget" name="QProgramUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QProgramUI</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QProgramTableWidget" name="m_pProgramTableWidget">
     <property name="minimumSize">
      <size>
       <width>780</width>
       <height>520</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>指令名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>指令参数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>备注</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>调试</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="m_pFrameToolButtomContainter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>520</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QToolButton" name="m_ptbInsertRow">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>插入一行</string>
        </property>
        <property name="text">
         <string>插 入</string>
        </property>
        <property name="icon">
         <iconset>
          <disabledoff>:/RightKeyMenuIcon/insert_enable</disabledoff>
          <disabledon>:/RightKeyMenuIcon/insert_disable</disabledon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="m_ptbUndo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>撤 销</string>
        </property>
        <property name="icon">
         <iconset>
          <disabledoff>:/RightKeyMenuIcon/undo_enable</disabledoff>
          <disabledon>:/RightKeyMenuIcon/undo_disable</disabledon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="m_ptbAddRow">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>添 加</string>
        </property>
        <property name="icon">
         <iconset>
          <disabledoff>:/RightKeyMenuIcon/add_enable</disabledoff>
          <disabledon>:/RightKeyMenuIcon/add_disable</disabledon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="m_ptbRedo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>重 做</string>
        </property>
        <property name="icon">
         <iconset>
          <disabledoff>:/RightKeyMenuIcon/redo_enable</disabledoff>
          <disabledon>:/RightKeyMenuIcon/redo_disable</disabledon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QToolButton" name="m_ptbCopy">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>复 制</string>
        </property>
        <property name="icon">
         <iconset>
          <disabledoff>:/RightKeyMenuIcon/copy_enable</disabledoff>
          <disabledon>:/RightKeyMenuIcon/copy_disable</disabledon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="m_ptbCut">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>剪 切</string>
        </property>
        <property name="icon">
         <iconset>
          <disabledoff>:/RightKeyMenuIcon/cut_enable</disabledoff>
          <disabledon>:/RightKeyMenuIcon/cut_disable</disabledon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QToolButton" name="m_ptbPaste">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>粘 贴</string>
        </property>
        <property name="icon">
         <iconset>
          <disabledoff>:/RightKeyMenuIcon/paste_enable</disabledoff>
          <disabledon>:/RightKeyMenuIcon/paste_disable</disabledon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="m_ptbDelete">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>删 除</string>
        </property>
        <property name="icon">
         <iconset>
          <disabledoff>:/RightKeyMenuIcon/delete_enable</disabledoff>
          <disabledon>:/RightKeyMenuIcon/delete_disable</disabledon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QToolButton" name="m_ptbSave">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>保 存</string>
        </property>
        <property name="icon">
         <iconset>
          <disabledoff>:/RightKeyMenuIcon/save_enable</disabledoff>
          <disabledon>:/RightKeyMenuIcon/save_disable</disabledon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QToolButton" name="m_ptbSaveAs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>另存为</string>
        </property>
        <property name="icon">
         <iconset>
          <disabledoff>:/RightKeyMenuIcon/saveAs_enable</disabledoff>
          <disabledon>:/RightKeyMenuIcon/saveAs_disable</disabledon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QToolButton" name="m_ptbCheck">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>检 查</string>
        </property>
        <property name="icon">
         <iconset>
          <disabledoff>:/RightKeyMenuIcon/check_enable</disabledoff>
          <disabledon>:/RightKeyMenuIcon/check_disable</disabledon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QToolButton" name="m_ptbOpen">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>打 开</string>
        </property>
        <property name="icon">
         <iconset>
          <disabledoff>:/RightKeyMenuIcon/open_enable</disabledoff>
          <disabledon>:/RightKeyMenuIcon/open_disable</disabledon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QProgramTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">../../../../QCustomModelViewLib/QProgramTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QCustomModelViewQrc.qrc"/>
 </resources>
 <connections/>
</ui>
